src/components/grapher/Grapher.vue

```diff
@@ -3,12 +3,17 @@
     <svg class="grapher-svg"><g> <!-- Note: we wrap the entire svg in a g so that svgPanZoom doesn't
             have to create a new g and Vue loses track of the DOM elements. -->
       <!-- Note:Inside svg, 1px = 1 eight-to-five step -->
-      <g class="grapher--line-container">
-        <rect
-          class="grapher--field-rect"
-          :width="fieldWidth"
-          :height="fieldHeight"
-        />
+
+    <rect
+      class="grapher--field-rect"
+      :width="fieldWidth"
+      :height="fieldHeight"
+      data-test="grapher--field-rect"
+    />
+      <g 
+        v-if="yardlines"
```

Comment:
Can you not affect the hash marks when the yardlines are disabled? We can file a new issue just for hash marks.

Reply:
sounds good I'll make separate wrapper for the hashmarks